src/Components/DynamicFields.res

```diff
@@ -493,13 +493,14 @@ let make = (
       ->React.array}
       <RenderIf condition={isRenderDynamicFieldsInsideBilling}>
         <div
-          className={`${spacedStylesForBiilingDetails} w-full text-left`}
+          className={`billing__section ${spacedStylesForBiilingDetails} w-full text-left`}
```

Comment:
Lets try to use `hypen-separated` class naming conventions as per mentioned in [adding-custom-styles](https://tailwindcss.com/docs/adding-custom-styles#using-css-and-layer).

Reply:
updated